components/Header.js

```diff
@@ -34,22 +40,24 @@ export default function Header(props) {
     setModalVisible(true);
   };
 
-  const headerHeight = 70;
+  const headerHeight = 80;
 
-  const logoAspect = 264 / 243;
-  const logoWidth = 90;
-  const logoHeight = logoWidth / logoAspect;
-  const logoTop = headerHeight - logoHeight / 2;
+  // const logoAspect = 264 / 243;
+  const logoWidth = 132;
+  const logoHeight = 122 // logoWidth / logoAspect;
+  const logoTop = headerHeight - logoHeight / 2 + 21;
```

Comment:
got you about image aspect:) I also think wrapping all image assets into easier component with 1:1 rendering